### PR TITLE
[RA1 Ch01, 02, 03, 04, 06, 07, 08] External references 

### PR DESCRIPTION
**Pull request by karinesevilla**
refs.bib file updated with external references.
References citations added into chapters 1, 2, ,3

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 1]
```diff
@@ -1,167 +1,167 @@
...
+Reference Architecture document can be used by operators to deploy
+Anuket conformant infrastructure; hereafter, “conformant” denotes that
+the resource can satisfy tests conducted to verify conformance with this
+reference architecture.
+
+Use Cases
+---------
+
+Several NFV use cases are documented in OpenStack. For more examples and
+details refer to the
```
Comment (ildikov) [inline doc/ref_arch/openstack/chapters/chapter01.rst:41]: There's a trailing whitespace here. This line and the next would probably fit into one.

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 2]
```diff
@@ -1,167 +1,167 @@
...
+Infrastructure Manager (VIM) chosen based on the criteria laid out in
+the :doc:`ref_model:chapters/chapter01`.
+OpenStack :cite:p:`openstack` has the advantage of being a
+mature and widely accepted open-source technology; a strong ecosystem of
+vendors that support it, the OpenInfra Foundation for managing the
+community, and, most importantly, it is widely deployed by the global
+operator community for both internal infrastructure and external facing
+products and services. This means that the operators have existing staff
+with the right skill sets to support a Cloud Infrastructure
+(or Network Function Virtualisation Infrastructure,
```
Comment (ildikov) [inline doc/ref_arch/openstack/chapters/chapter01.rst:17]: There's a trailing whitespace here, would be good to remove.

**Review by ildikov — APPROVED** [review]
Hi Karine,

Thank you for fixing the references. My comments in line are more wishlist itms in case you need to upload a new revision.

File: doc/ref_arch/openstack/chapters/chapter01.rst (hunk 1 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter01.rst]: @ildikov Thanks! Trailing white space removed

File: doc/ref_arch/openstack/chapters/chapter01.rst (hunk 2 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter01.rst]: @ildikov Thanks! Removed

**Comment by karinesevilla** [comment]
@ildikov Chapters 7 and 8 are ready to be reviewed

**Comment by pgoyal01** [comment]
@karinesevilla @ildikov I have added the external references for Ch04.

**Comment by gkunz** [comment]
I noticed that the files of chapters 1, 2 and 3 are using Windows line ending, whereas the other files use Unix file endings. As a result, when building using tox, doc8 complains about wrong line endings for those files. Can you please convert them back to Unix line endings?

File: doc/ref_arch/openstack/chapters/chapter02.rst [hunk 3]
```diff
@@ -1,2264 +1,2253 @@
...
+     - :ref:`chapters/chapter06:workload security`
+   * - sec.sys.009
+     - Access
+     - The Platform **must** support creation of Trust Relationships between
+       trust domains. These maybe uni-directional relationships where the
+       trusting domain trusts another domain (the "trusted domain") to
+       authenticate users for them them or to allow access to its resources
+       from the trusted domain. In a bidirectional relationship both domain
+       are "trusting" and "trusted"
+     - :ref:`chapters/chapter04:logical segregation
```
Comment (gkunz) [inline doc/ref_arch/openstack/chapters/chapter02.rst:657]: Remove trailing white space.

```suggestion
     - :ref:`chapters/chapter04:logical segregation

```

File: doc/ref_arch/openstack/chapters/chapter02.rst [hunk 4]
```diff
@@ -1,2264 +1,2253 @@
...
+       to support various infrastructure profiles (Basic and High
+       Performance)
+     - :ref:`chapters/chapter03:\
+       cloud partitioning: host aggregates, availability zones`
+   * - inf.com.08
+     - Compute
+     - The Architecture **must** support allocating certain number of host
+       cores for all non-tenant workloads such as for OpenStack services.
+       SMT threads can be allocated to individual OpenStack services or their
+       components. Dedicating host cores to certain
```
Comment (gkunz) [inline doc/ref_arch/openstack/chapters/chapter02.rst:1331]: remove trailing whitespace.
```suggestion
       components. Dedicating host cores to certain

```

File: doc/ref_arch/openstack/chapters/chapter02.rst [hunk 5]
```diff
@@ -1,2264 +1,2253 @@
...
+   * - sec.std.001
+     - Standards
+     - The Cloud Operator **should** comply with Center for Internet Security
+       CIS Controls :cite:p:`cis`
+     -
+   * - sec.std.002
+     - Standards
+     - The Cloud Operator, Platform and Workloads **should** follow the
+       guidance in the CSA Security Guidance for Critical Areas of Focus in
+       Cloud Computing (latest version)- CSA,
```
Comment (gkunz) [inline doc/ref_arch/openstack/chapters/chapter02.rst:2194]: Remove trailing whitespace.
```suggestion
       Cloud Computing (latest version)- CSA,

```

File: doc/ref_arch/openstack/chapters/chapter03.rst [hunk 6]
```diff
@@ -1,877 +1,874 @@
...
+
+:numref:`Network Fabric – Physical` shows a physical network layout where each
+physical server is dual homed to TOR (Leaf/Access) switches with redundant
+(2x) connections. The Leaf switches are dual homed with redundant connections
+to spines.
+
+Storage
+^^^^^^^
+
+OpenStack supports many different storage architectures and
```
Comment (gkunz) [inline doc/ref_arch/openstack/chapters/chapter03.rst:689]: Remove trailing whitespace
```suggestion
OpenStack supports many different storage architectures and

```

**Review by gkunz — CHANGES_REQUESTED** [review]

**Review by ildikov — COMMENTED** [review]
It looks good to me beyond Georg's comments.

Thanks,
Ildikó

File: doc/ref_arch/openstack/chapters/chapter02.rst (hunk 3 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter02.rst]: @gkunz Thanks! Removed

File: doc/ref_arch/openstack/chapters/chapter02.rst (hunk 4 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter02.rst]: @gkunz Thanks! Removed

File: doc/ref_arch/openstack/chapters/chapter02.rst (hunk 5 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter02.rst]: @gkunz Thanks! Removed

File: doc/ref_arch/openstack/chapters/chapter03.rst (hunk 6 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter03.rst]: @gkunz Thanks! Removed

**Review by pgoyal01 — APPROVED** [review]

File: doc/ref_arch/openstack/refs.bib [hunk 7]
```diff
@@ -1,108 +1,829 @@
...
+}
+
+@misc{etsisol4,
+	title = {{Network Functions Virtualisation (NFV) Release 4; Protocols and Data Models; VNF Package and PNFD Archive specification}},
+	howpublished = {ETSI GS NFV-SOL 004 V4.3.1},
+	publisher =	{ETSI},
+	url = {https://www.etsi.org/deliver/etsi_gs/NFV-SOL/001_099/004/04.03.01_60/gs_NFV-SOL004v040301p.pdf},
+    author = {},
+	year = 2022,
+	month =	 July
```
Comment (gkunz) [inline doc/ref_arch/openstack/refs.bib:764]: Replacing full month with keyword

```suggestion
	month =	 jul

```

File: doc/ref_arch/openstack/refs.bib [hunk 8]
```diff
@@ -1,108 +1,829 @@
...
+}
+
+@misc{etsisec21,
+	title = {{Network Functions Virtualisation (NFV) Release 2; Security; VNF Package Security Specification}},
+	howpublished = {ETSI GS NFV-SEC 021 V2.6.1},
+	publisher =	{ETSI},
+	url = {https://www.etsi.org/deliver/etsi_gs/NFV-SEC/001_099/021/02.06.01_60/gs_nfv-sec021v020601p.pdf},
+    author = {},
+	year = 2019,
+	month =	 June
```
Comment (gkunz) [inline doc/ref_arch/openstack/refs.bib:774]: Replacing full month with keyword:

```suggestion
	month =	 jun

```

File: doc/ref_arch/openstack/refs.bib [hunk 9]
```diff
@@ -1,108 +1,829 @@
...
+}
+
+@misc{openstackcind,
+	title = {{Cinder Driver Support Matrix}},
+	url = {https://docs.openstack.org/cinder/latest/reference/support-matrix.html},
+    author = {OpenStack},
+	note = {Accessed: 2022-07-25}
+}
+
+@misc{tungst,
```
Comment (gkunz) [inline doc/ref_arch/openstack/refs.bib:302]: Wrong key referenced from chapter 03.

```suggestion
@misc{tungsten,

```

**Review by gkunz — CHANGES_REQUESTED** [review]
While building locally using tox, I encountered some additional "bugs", with proposed fixed below.

File: doc/ref_arch/openstack/refs.bib [hunk 10]
```diff
@@ -1,108 +1,829 @@
...
+
+% Chapter 2 References
+
+@techreport{cispwd,
+	title =	{{CIS Password Policy Guide}},
+	publisher =	{Center for Internet Security},
+	url = {https://www.cisecurity.org/insights/white-papers/cis-password-policy-guide},
+	author = {H. Philip White},
+	year = 2020
+}
```
[the remark this answers is not in the document]
Comment (karinesevilla) [reply doc/ref_arch/openstack/refs.bib:158]: @gkunz Thanks!

**Comment by karinesevilla** [comment]
@gkunz Thanks for the check with tox. All proposals are accepted. Ready for another review

**Comment by gkunz** [comment]
Hi @karinesevilla. Thank you for including all comments. The last outstanding item is the conversion of chapter 1-3 from Windows back to Unix line endings. Do you use an editor you can use for this? Otherwise I can try to push an update myself.

**Review by gkunz — APPROVED** [review]
Perfect! Thanks for the updates!

**Comment by karinesevilla** [comment]
Hi @gkunz I've made modifications to change the end of lines from Windows to Unix ( CRLF to LF) via VSCode locally. I am not totally sure it will work. Can you check? If it does not work, can you update it directly?

**Pull request merged**